sauron/sauron.py

```diff
@@ -38,11 +34,12 @@ def fetch(url):
 
     return session.get(url)
 
-
 @plugin.init()
-def init(plugin, options, configuration, **kwargs):
-    plugin.api_endpoint = options["sauron-api-endpoint"]
-    if not plugin.api_endpoint:
+def init(plugin, options, **kwargs):
+    plugin.sauron_api_endpoint = options.get("sauron-api-endpoint", None)
```

Comment:
Why rename this variable?

Reply:
I renamed it because I saw `options["sauron-api-endpoint"]` and I thought the variable name needed to match that. Now I understand the options specify sauron for the user to know they are related to sauron since its running on core lightning but there was no need to change the variable name since that is only in the sauron code. Will revert back those changes.